dashboard/15-final/app/lib/calculations.tsx

```diff
@@ -38,7 +38,7 @@ export const countCustomerInvoices = (
   invoices: Invoice[],
   customerId: number,
 ) => {
-  return invoices.filter((invoice) => invoice.customerId === customerId).length;
+  return invoices.filter((invoice) => invoice.customer_id === customerId).length;
 };
 
 export const findLatestInvoices = (invoices: Invoice[]) => {
```

Comment:
`findLatestInvoices`, this is another case where we can optimize using SQL, there's a rough comment above with how we can get only the last 5 invoices using SQL.